llvm/lib/Transforms/Vectorize/VPlanTransforms.h

```diff
@@ -183,6 +183,10 @@ struct VPlanTransforms {
   /// CanonicalIVTy as type for all un-typed live-ins in VPTypeAnalysis.
   static void simplifyRecipes(VPlan &Plan, Type &CanonicalIVTy);
```

Comment:
Strange: I thought simplifyRecipes was a static function. (Can address in follow-up)

Reply:
Ah, simplifyRecipe versus Recipes. Never mind.

Reply:
It is but it's also called from LoopVectorizer.cpp, unlike the other passes in VPlanTransforms::optimize